displays/cavity_display/backend/runner.py

```diff
@@ -1,12 +1,14 @@
+import sys
 from datetime import datetime
 from time import sleep
 from typing import List, Optional
 
 from lcls_tools.common.controls.pyepics.utils import PV
 
-from displays.cavity_display.backend.backend_cavity import BackendCavity
-from displays.cavity_display.utils.utils import DEBUG, BACKEND_SLEEP_TIME
-from utils.sc_linac.linac import Machine
+sys.path.append("/home/physics/srf/sc_linac_physics")
```

Comment:
Ooh, will we need to modify this line when we run runner.py locally if our local setup doesn't have this exact directory path setup? (just curious, not a comment on a problem)

Reply:
Oh no, locally you can just edit your own python path. This will still be added to it but it won't cause any issues. The problem here was that I couldn't figure out how to set the python path in a different server lol Since it launches the program in srv03

Reply:
Ah okay!